src/bosh_azure_cpi/lib/cloud/azure/cloud.rb

```diff
@@ -595,6 +595,14 @@ def set_disk_metadata(disk_id, metadata)
       raise Bosh::Clouds::NotImplemented
     end
 
+   # Information about the CPI
+   # @return [Hash] CPI properties
+   def info
+     {
+       'stemcell_formats' => %w(azure-vhd azure-light)
+     }
+   end
+
```

Comment:
What's the behavior of the Director based on the `stemcell_formats`?
For Azure CPI, the heavy stemcell is used for Linux, and the light stemcell should be used for Windows.
Would the Director try to upload a light stemcell for linux? In fact, such a stemcell doesn't exist.